views/pages/practitioner/declaration.njk

```diff
@@ -0,0 +1,354 @@
+                    {{ __('edit') }}
+                </a>
+            </div>
+        </summary>
+
+        <table class="table-fixed">
+            <tbody class="bg-gray-100">
+                <tr class="odd:bg-gray-300">
+                    <th scope="row" class="w-1/3 text-left font-normal p-4 border-2 border-white align-top">{{ __('declaration.you_are_a') }}</th>
+                    <td class="w-2/3 p-4 border-2 border-white align-top">
+                        {{ data.practitionerTypeText }}
+                    </td>
+                </tr>
+                <tr class="odd:bg-gray-300">
+                    <th scope="row" class="w-1/3 text-left font-normal p-4 border-2 border-white align-top">{{ __('Name') }}</th>
+                    <td class="w-2/3 p-4 border-2 border-white align-top">
+                        {{ data.practitionerFullName }}
+                    </td>
+                </tr>
+                <tr class="odd:bg-gray-300">
+                    <th scope="row" class="w-1/3 text-left font-normal p-4 border-2 border-white align-top">{{ __('Address') }}</th>
+                    <td class="w-2/3 p-4 border-2 border-white align-top">
+                        {{ data.practitionerAddress }}<br>
+                        {{ data.practitionerCity }}, {{ data.practitionerProvince }}<br>
+                        {{ data.practitionerPostal }}
+                    </td>
+                </tr>
+                <tr class="odd:bg-gray-300">
+                    <th scope="row" class="w-1/3 text-left font-normal p-4 border-2 border-white align-top">{{ __('Email') }}</th>
+                    <td class="w-2/3 p-4 border-2 border-white align-top">
+                        {{ data.practitionerEmail }}
+                    </td>
+                </tr>
+                <tr class="odd:bg-gray-300">
+                    <th scope="row" class="w-1/3 text-left font-normal p-4 border-2 border-white align-top">{{ __('phone_number') }}</th>
+                    <td class="w-2/3 p-4 border-2 border-white align-top">
+                        {{ data.practitionerPhone }}
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+    </details>
+
+    <form action="{{ route('declaration.store') }}" method="post" class="w-full">
+        <div class="mt-8 text-lg">
+            <p class="font-bold">{{ __('declaration.i_confirm_that') }}</p>
+        </div>
+
+        {{ signaturePad() }}
+
+        <div class="md:w-1/2">
+            <button type="submit">{{ __('declaration.submit_to_service_canada') }}</button>
```

Comment:
Can we add `class="bg-green-700"` to the big submit buttons? For the most part we use a blue call to action button, but where we have two actions, or it's the big submit, we're trying green.

Reply:
done